dask/dataframe/reshape.py

```diff
@@ -239,8 +239,9 @@ def melt(frame, id_vars=None, value_vars=None, var_name=None,
          value_name='value', col_level=None):
 
     from dask.dataframe.core import no_default
+    from ..utils import M
```

Comment:
In general this looks good to me, but if possible it would be good to move this import up to the top of the file.

Reply:
Sure. Would you prefer the already existing `from dask.dataframe.core import no_default` to be moved up as well?

Reply:
I suspect that that is here because placing it at the top would cause a circular import (core.py probably imports reshape.py)

Reply:
We shouldn't run into the same difficulty with utils.py